### PR TITLE
fix(Tab2): `Tabs` should be output with `type="button"` to prevent accidental form submission

### PR DESCRIPTION
**Pull request by lukelooker**
Addresses a similar issue with `ListItemContent` as well.

## Developer Checklist [ℹ️](https://github.com/looker-open-source/components/blob/main//CONTRIBUTING.md#developer-checklist)

- [x] ♿️ Accessibility addressed
- [x] 🖼 Image Snapshot coverage

### CODE REVIEW

File: packages/components/src/ListItem/ListItemContent.tsx [hunk 1]
```diff
@@ -40,7 +40,9 @@ import type {
 } from './types'
 import { listItemBackgroundColor, listItemPaddingX } from './utils'
 
-const Button = styled.button`
+const Button = styled.button.attrs(({ type = 'button' }) => ({
+  type,
+}))`
```
Comment (ElliotTheHuman) [inline packages/components/src/ListItem/ListItemContent.tsx:45]: Didn't know that the default `type` for `button` was "submit" 🤷 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

Good call.

**Review by ElliotTheHuman — DISMISSED** [review]
LGTM

**Comment by lukelooker** [comment]
Added test coverage

**Review by ElliotTheHuman — APPROVED** [review]
👍 👍

**Pull request merged**